Drv/TcpClient/TcpClientComponentImpl.cpp

```diff
@@ -0,0 +1,78 @@
+// ======================================================================
```

Comment:
Is there some way we can make it obvious that the TcpClient, TcpServer, and Udp are all ByteStreamManager implementations?

There's not any obvious links within the code of the components themselves, so I was surprised to see components without ComponentAi.xml files.

Reply:
It will be part of the SDD.  The code doesn't care what model it is implementing, it just does.